bedrock/firefox/templates/firefox/new/protocol/download.html

```diff
@@ -0,0 +1,19 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/quantum" %}
+
+{% extends "firefox/new/protocol/base_download.html" %}
+
+{% block experiments %}{% endblock %}
+
+{% block site_header %}{% endblock %}
```

Comment:
Should this be in the base template? The platform pages are showing the nav, which isn't what we currently have in prod. I understand there have been experiments related to this, but afaik it didn't turn out in favor?

Reply:
There was a request to add it to the platform pages specifically.

You are right though, the tests on /new did not go well. 

I guess I'll sleep on what the right thing to do here is.